argo/kfp-compiler/poetry.lock

```diff
@@ -1,100 +1,186 @@
+# This file is automatically @generated by Poetry 1.7.1 and should not be changed by hand.
```

Comment:
Poetry 1.7.1? It feels odd since we use 1.5.1... but I imagine it's backwards compatible?

Reply:
I had to reinstall Poetry. I recreated the lock file. I forgot abt this, but yes, compstible. Can change if you prefer, if I can get 1.5.1 back and working locally.

Reply:
Maybe we should just upgrade to Poetry 1.7.1 in `.tool-versions`?

Reply:
done